src/vellum/workflows/nodes/displayable/web_search_node/node.py

```diff
@@ -1,9 +1,17 @@
+import logging
 from typing import Any, ClassVar, Dict, List, Optional
 
+from requests import Request, RequestException, Session
+from requests.exceptions import JSONDecodeError
+
+from vellum.workflows.errors.types import WorkflowErrorCode
+from vellum.workflows.exceptions import NodeException
 from vellum.workflows.nodes.bases import BaseNode
 from vellum.workflows.outputs import BaseOutputs
 from vellum.workflows.types.generics import StateType
 
+logger = logging.getLogger(__name__)
+
 
 class WebSearchNode(BaseNode[StateType]):
```

Comment:
[oos] Let's get a `tests/workflows` or `src/vellum/workflows/nodes/displayable/web_search_node/tests/test_node.py` just to show that this is working e2e. We'll want to mock out the requests library and ensure we are sending the correct body and handling expected responses correctly. `requests_mock` is useful for this